Makefile

```diff
@@ -203,24 +203,44 @@ lint-fix:
 
 SIMAPP = ./tests/simulation
 
-test-sim-non-determinism:
+# Install the runsim binary
+runsim: $(RUNSIM)
+$(RUNSIM):
+	@echo "Installing runsim..."
+	@go install github.com/cosmos/tools/cmd/runsim@v1.0.0
+
+test-app-non-determinism:
 	@echo "Running non-determinism application simulations..."
 	@go test -mod=readonly $(SIMAPP) -run TestAppStateDeterminism -Enabled=true \
 		-NumBlocks=100 -BlockSize=200 -Commit=true -Period=0 -v -timeout 24h
 
-test-sim-multi-seed-short:
+test-app-multi-seed-short: runsim
 	@echo "Running short multi-seed application simulations. This may take a while!"
 	@runsim -Jobs=4 -SimAppPkg=$(SIMAPP) -ExitOnFail 50 10 TestFullAppSimulation
 
-test-sim-benchmark-invariants:
+test-sim-multi-seed-long: runsim
+	@echo "Running long multi-seed application simulation. This may take awhile!"
+	@runsim -Jobs=4 -SimAppPkg=$(SIMAPP) -ExitOnFail 500 50 TestFullAppSimulation
+
+test-app-import-export: runsim
+	@echo "Running application import/export simulation. This may take several minutes..."
+	@runsim -Jobs=4 -SimAppPkg=$(SIMAPP) -ExitOnFail 50 5 TestAppImportExport
+
+test-app-after-import: runsim
```

Comment:
`runsim` is already installed during one of the CI steps. We also don't need to query the registry twice. Maybe we can remove runsim as a dependency for this job (and the jobs above)? I think it's not needed.

Reply:
It is need for running tests individually